lib/cfg-grammar.y

```diff
@@ -895,6 +897,15 @@ template_content
         : { last_template = log_template_new(configuration, NULL); } template_content_inner	{ $$ = last_template; }
         ;
 
+template_content_list
+        : template_content_list_build                     { $$ = $1; }
+        ;
+
+template_content_list_build
```

Comment:
I think you can omit the `template_content_list_build` non-terminal symbol and just use `template_content_list`:

```
template_content_list : template_content template_content_list { $$ = prepend(); } | { $$ = NULL; }
```

Reply:
Done thanks!